indico/modules/events/registration/templates/management/regform_edit.html

```diff
@@ -29,4 +29,43 @@
             {% trans %}Back{% endtrans %}
         </a>
     {% endcall %}
+
+    {% if regform %}
+        <script>
+            (function () {
+                'use strict';
+
+                $('#message-pending-preview').attr('data-href', '{{ url_for('.notification_preview', regform) }}').ajaxDialog({
```

Comment:
When assigning the field descriptions at runtime (in the form's constructor), you would avoid having to use JS to set the `data-href` attributes :)

Reply:
And how'd you do it if the URL to point to is dynamically generated and based on the regform?

Reply:
The form's constructor runs while a request is being handled, so you have all the necessary data available :)

Reply:
Yep, I was about to tell you I was just checking the constructor. Thanks for the tip 🙏

Reply:
Changed 👍